src/darwintracelib1.0/darwintrace.c

```diff
@@ -632,51 +683,116 @@ static inline bool __darwintrace_sandbox_check(const char *path, int flags) {
 
 	char command = -1;
 	char *t;
+    bool inserted_in_dtsm, path_permission, is_prefix;
+
+    /*
+     * insertions to shared memory cache need not be always successful.
+     * It only allows most frequently occuring characters.
+     * Also, insertions are unsuccessful in rare cases and if that's not the case
+     * changes should be made.
+     * See LOWER_LIMIT, UPPER_LIMIT & POSSIBLE_CHARACTERS in dtsharedmemory.h
+     */
+
```

Comment:
I will correct it. I never noticed that because while working on my pc, the code seemed to have consistent indentation (sorry first time encountered such a situation).